### DISCUSSION TITLE
getting dependencies when building from source

### DISCUSSION THREAD

**Discussion by davidgritter** [GrandOrgue/grandorgue#1091] [Help] [answered]
I regularly build G.O. from source, but in the latest download, BUILD.md asks to download four archives (rtaudio, rtmidi, zita-convolver, and port audio)  However, none of the URLs provided in this file appear to work.  In particular, the zita-convolver link to salsa.debian.org requires a login, and the others return a 404 error.  would'nt it be better to provide a link directly to 
https://kokkinizita.linuxaudio.org/linuxaudio/downloads/index.html for zita-convolver, and links to the code pages on github where a zip file download can be requested for the other archives?

**Comment by rousseldenis** [comment, marked as answer]
@davidgritter Maybe this has solved your issue ?

https://github.com/GrandOrgue/grandorgue/commit/2b983c21821c0daa2c0842bca17dd833c4017d8a

**Reply by davidgritter** [reply]
My intent was to address the fact that none of the links in the Build.md were working properly.  I have recently checked, and they are now all working properly, including the link to salsa.debian.org for zita-convolver

**Reply by rousseldenis** [reply]
Ok, so you wanted to say Build.md was wrong in former versions, but now it has been solved ?
